tutorials/quantum_oracle/quantum_oracle_tutorial.ipynb

```diff
@@ -0,0 +1,183 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #8.        apply_to_all(H,x)
</pre><p>cold this line and the two ones that comes afterwords be combined as "within_apply" statement? If so, I would have implement it this way (maybe as separate function), add to the "classiq concepts" list above with the links.Also add under the function of with the "within_apply" few word on this concept (compute and uncompute, freeing qubits for further use).</p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Classiq/classiq-library/pull/186/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>